build.gradle

```diff
@@ -3,11 +3,12 @@ apply from: 'checkstyle.gradle'
 
 buildscript {
     repositories {
+        google()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:3.0.0'
         classpath 'com.github.ben-manes:gradle-versions-plugin:0.14.0'
         classpath 'org.codehaus.groovy:groovy-android-gradle-plugin:1.2.0'
```

Comment:
@friederbluemle if @dariuszseweryn is right, maybe temporary changing to `classpath 'org.codehaus.groovy:groovy-android-gradle-plugin:1.3.0-SNAPSHOT'` would fix groovy compilation errors

Reply:
Good idea, you could definitely give it a try! Ultimately it's up to the maintainers of `RxAndroidBle` if they want a snapshot dependency on the `master` branch.
If the update does fix the issue and this is the only problem, maybe we could nudge `groovy-android-gradle-plugin` to release an official new version soon.

Reply:
I tried it locally. However, the test did not pass. I think that the change has not been uploaded in the snapshot yet.

http://oss.jfrog.org/oss-snapshot-local/org/codehaus/groovy/groovy-android-gradle-plugin/

> 1.3.0-SNAPSHOT/      07-Jun-2017 17:25